### ISSUE TITLE
history trade data can generate renko?

### ISSUE THREAD

**Issue by huangxiancun** [aticio/renko#2]
can you have good idea?

**Comment by aticio** [comment]
I'm not sure I understand correctly but, you can generate a renko chart with historical market data using this library. The library cannot create visual graphics but you can generate the data (open and close prices and direction of renko bricks).

**Comment by huangxiancun** [comment]
Thanks，i see  how to using this library. k line data can not explain "maoci" phenomenon. pls，you can see this picture with bule box.
![image](https://user-images.githubusercontent.com/29544871/147635448-bb4d6f83-17c7-4f03-9b62-da10354aeffd.png)

**Comment by aticio** [comment]
Okay, got it. This feature is not yet available. I will add it as soon as possible.

**Comment by huangxiancun** [comment]
666, if this library can supporting history trade data  input, this is something meaningful. i also do it haha.
![image](https://user-images.githubusercontent.com/29544871/147637414-d924afdb-4c59-43f0-bd10-93af62d94e1d.png)

**Comment by aticio** [comment]
Working on it.

**Comment by huangxiancun** [comment]
i see you code carefully,  your code can support history trade data.  but you can solve "maoci" phenomenon. it is perfect . haha

**Comment by huangxiancun** [comment]
The accuracy problem can be solved by using method “decimal” library.
![image](https://user-images.githubusercontent.com/29544871/147648433-b066ae8a-fadb-4fab-8400-ae254b9e4830.png)

**Comment by huangxiancun** [comment]
you can add date of some kline data in bricks data.this infomation can confirme generate each bricks rightly. this is important,especially you analyse loss orders.

**Comment by aticio** [comment]
Ok, I updated the code. The wicks are available in the output as "low" or "high" values. Also the decimals are reduced to two digits. I also added a test case "test_checkrenko.py". This test is for checking the accuracy of bricks and wicks. It compares the open, high, low and close values with tradingview's renko chart. For that, you must give first initial values manually. Here is the code description:
(https://gist.github.com/aticio/604d5a6971fa79e469793dedd482b886)
